.github/workflows/android.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           api-level: ${{ matrix.api-level }}
           target: ${{ matrix.target }}
           arch: x86_64
-          script: ./gradlew connectedCheck --no-daemon --stacktrace -PuseKsp=false
+          script: ./gradlew connectedCheck --no-daemon --stacktrace
```

Comment:
Why was this needed?

Didn't we figure out a way to avoid running something on kapt in tests directly as opposed to skipping it here?

Reply:
Yeah I thought so as well. After watching the CI build that failed it looked like it was building KSP tests at that stage. Removing this flag made it run the KAPT version at least. I was suspecting that it checked if we had that flag, not that it was set to true. I can remove this change ofc, but then I am unsure if it will pass CI.

Reply:
I think that we actually need this change from testing this locally. I think it is since we check `project.hasProperty('useKsp')` and that checks that we have the property, not that it is true.